src/tasks/TaskHelper.ts

```diff
@@ -62,45 +59,39 @@ export interface DockerRunTaskContext extends DockerTaskExecutionContext {
 // This doesn't need to be extended so redefining for parity is simplest
 export type DockerComposeTaskContext = DockerTaskExecutionContext;
 
-export interface TaskHelper {
+export abstract class TaskHelper {

```

Comment:
We'll need a "DefaultTaskHelper" or similar that will be used if there isn't a specific project type TaskHelper (dotnet, python, etc.) since build is now part of the TaskHelper. Otherwise we won't be able to build anything but .NET, Node, and Python projects. Basically the helper should be one of the specific helpers if one applies, otherwise it should be a default/generic helper.